.buildkite/ftr_configs.yml

```diff
@@ -399,3 +399,4 @@ enabled:
   - x-pack/performance/journeys/dashboard_listing_page.ts
   - x-pack/performance/journeys/cloud_security_dashboard.ts
   - x-pack/test/custom_branding/config.ts
+  - x-pack/test/profiling_api_integration/cloud/config.ts
```

Comment:
__nit:__ don't forget to update CODEOWNERS file to make sure your tests aren't updated without your review

Reply:
https://github.com/elastic/kibana/pull/159984/commits/52a5ef4d85385f07ac2a51b67c404021932afabc

Reply:
ah, interesting, I recall I had added it before, and I just noticed the CI removes it automatically https://github.com/elastic/kibana/pull/159984/commits/10a155d50c8399a97cc12c86b65dc97d87c22002

Reply:
I don't think so, I had added it before here and CI removed it anyways: https://github.com/elastic/kibana/pull/159984/commits/bc65a1882a1a8b64b9fe350f9bfc33cf1251ccf8#diff-3d36a1bf06148bc6ba1ce2ed3d19de32ea708d955fed212c0d27c536f0bd4da7R530

Reply:
got it... https://github.com/elastic/kibana/pull/159984/commits/ac00a3234572b8d6605f2054e26c737a8360aed2